packages/typegpu/src/core/constant/tgpuConstant.ts

```diff
@@ -0,0 +1,66 @@
+import type { AnyWgslData, Infer } from '../../data';
```

Comment:
Tiny nit, let's refrain from importing from aggregate files in internal code.
```suggestion
import type { AnyWgslData } from '../../data/wgslTypes';
import type { Infer } from '../../shared/repr';
```